### ISSUE TITLE
No license in pom file

### ISSUE THREAD

**Issue by ganadist** [commonmark/commonmark-java#347]
I'm using [Google OSS License plugin](https://github.com/google/play-services-plugins/tree/main/oss-licenses-plugin#readme) to show libraries' licenses to end users.

And after add `commonmark-java` dependency, I found license activity don't show license of `commommark-java`

I found there is license declaration on root (commonmark-parent) project.
https://github.com/commonmark/commonmark-java/blob/main/pom.xml#L264-L270

But, main java project(commonmark-parent) don't have license declaration.
https://github.com/commonmark/commonmark-java/blob/main/commonmark/pom.xml

And there is no license declaration on Maven POM File.
https://central.sonatype.com/artifact/org.commonmark/commonmark

Could you add license declaration on POM file of `commonmark-java`?

**Comment by ganadist** [comment]
I found https://maven.apache.org/pom.html#inheritance and it looks Google OSS License plugin does not implement inheritance case from parent.
I'll submit issue on https://github.com/google/play-services-plugins

**Comment by robinst** [comment]
Agreed that this is a bug in play-services-plugins, but I'm happy to add it to at least the core pom to work around it, see:
- https://github.com/commonmark/commonmark-java/pull/351

**Comment by ganadist** [comment]
Confirmed that license appeared on OSS License activity with commonmark-java 0.24.0
Thank you very much.
<img src="https://github.com/user-attachments/assets/a515661f-145c-4377-88c5-5028940d165a" width="340" >
